web/src/components/newsList.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import loadable from '@loadable/component';
+import { slidesToShowPlugin, arrowsPlugin } from '@brainhubeu/react-carousel';
+
+import NewsCard from './newsCard';
+import MoreButton from './moreButton';
+
+const Carousel = loadable(() => import('./carouselWrapper'));
+
+export default function NewsList({ news }) {
+    const content = {
+        text: 'More ❯',
+        back: '❮ Back',
+    };
+    return (
+        <div className="bg-brand-1 h-full mb-48 pb-48">
+            <h1 className="pt-12 text-center text-white text-4xl uppercase">
```

Comment:
Does the `header-accent` class work here?

Reply:
@asasmith yes